service/worker/deployment/fx.go

```diff
@@ -129,9 +134,9 @@ func (s *workerComponent) Register(registry sdkworker.Registry, ns *namespace.Na
 
 	// TODO Shivam: Might need a cleanup function upon activity registration
 	deploymentActivities := s.newDeploymentActivities(ns.Name(), ns.ID())
-	deploymentSeriesActivities := s.newDeploymentSeriesActivities(ns.Name(), ns.ID())
+	// deploymentSeriesActivities := s.newDeploymentSeriesActivities(ns.Name(), ns.ID())
 	registry.RegisterActivity(deploymentActivities)
-	registry.RegisterActivity(deploymentSeriesActivities)
+	// registry.RegisterActivity(deploymentSeriesActivities)
 	return nil
 }
```

Comment:
these changes also cherry-picked from https://github.com/temporalio/temporal/pull/6864/commits/8c856b829c38f073f6056d24022cea8f74da9845 to get the server to build -- will remove